dq-server/src/middleware/error.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function(err, req, res, next) {
+	res.status(500).send('Something Failed');
```

Comment:
In which cases is this called?

Reply:
The more specific the error is, the better

Reply:
Every time there is an exception in the request processing pipeline. In the future the logging logic must be refined.

Reply:
Can you put 'internal error' instead ?